generators/client/templates/react/src/main/webapp/swagger-ui/index.html.ejs

```diff
@@ -17,194 +17,70 @@
-        $(function() {
-            var springfox = {
-                "baseUrl": function() {
-                    var urlMatches = /(.*)\/swagger-ui\/index.html.*/.exec(window.location.href);
-                    return urlMatches[1];
-                },
-                "securityConfig": function(cb) {
-                    $.getJSON(this.baseUrl() + "/swagger-resources/configuration/security", function(data) {
-                        cb(data);
-                    });
-                },
-                "uiConfig": function(cb) {
-                    $.getJSON(this.baseUrl() + "/swagger-resources/configuration/ui", function(data) {
-                        cb(data);
-                    });
-                }
-            };
-            window.springfox = springfox;
-            window.oAuthRedirectUrl = springfox.baseUrl() + './dist/o2c.html'
+<body>
+<div id="swagger-ui"></div>
 
-            window.springfox.uiConfig(function(data) {
-                window.swaggerUi = new SwaggerUi({
-                    dom_id: "swagger-ui-container",
-                    validatorUrl: data.validatorUrl,
-                    supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
-                    onComplete: function(swaggerApi, swaggerUi) {
-                        initializeSpringfox();
-                        if (window.SwaggerTranslator) {
-                            window.SwaggerTranslator.translate();
-                        }
-                        $('pre code').each(function(i, e) {
-                            hljs.highlightBlock(e)
-                        });
-                    },
-                    onFailure: function(data) {
-                        log("Unable to Load SwaggerUI");
-                    },
-                    docExpansion: "none",
-                    apisSorter: "alpha",
-                    showRequestHeaders: false
-                });
+<script src="./swagger-ui-bundle.js"></script>
+<script src="./swagger-ui-standalone-preset.js"></script>
+<script src="./axios.min.js"></script>
 
-                initializeBaseUrl();
 
-                $('#select_baseUrl').change(function() {
-                    window.swaggerUi.headerView.trigger('update-swagger-ui', {
-                        url: $('#select_baseUrl').val()
-                    });
-                    addApiKeyAuthorization();
-                });
+<script type="text/javascript">
+    window.onload = function () {
+        var urls = [];
+        axios.get("/swagger-resources").then(function (response) {
+            response.data.forEach(function (resource) {
+                urls.push({"name": resource.name, "url": resource.location});
+            });
 
-                function addApiKeyAuthorization() {
-                <%_ if (authenticationType === 'session' || authenticationType === 'uaa') { _%>
-                    var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("X-XSRF-TOKEN", getCSRF(), "header");
```

Comment:
how do you handle this part now ?
Just tested your PR and it doesn't work for OAuth2 when I want to POST an entity

Reply:
you are right, I forgot to restore that part sorry. I fixed it now.

Reply:
thanks, just tell me if it's ready, so I can do some tests to be sure, before merging this

Reply:
Yes it should work now you can test it